src/forms/_base.scss

```diff
@@ -184,18 +184,27 @@ select {
 .form-field {
   position: relative;
   margin-bottom: ms(3);
+}
 
-  // @todo Turn this into a modifier when nightshade styles port over (e.g. form-field--standard)
-  &:not(.form-field--float) {
+.form-label {
+  @include heading-5;
+  display: block;
+  margin-bottom: ms(-6);
+  color: color(text);
 
-    .form-label {
-      @include heading-5;
-      display: block;
-      margin-bottom: ms(-6);
-      color: color(text);
+  & + .form-input {
+    margin-top: ms(-6);
+  }
 
-      & + .form-input {
-        margin-top: ms(-6);
+  &.form-label--mixed-case {
```

Comment:
That's fair - any thoughts? Some of my original thoughts were `less-emphasis`, `level-2`, `lower` ... do we like any of these better?